# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/libgovirt/hotfix-no-tests.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- libgovirt-v0.3.9/meson.build.vanilla	2024-04-28 18:05:26.575000000 +0200
+++ libgovirt-v0.3.9/meson.build	2024-04-28 18:05:36.786000000 +0200
@@ -93,7 +93,7 @@
 subdir('govirt')
 subdir('po')
 subdir('examples')
-subdir('tests')
+#subdir('tests')
 
 
 #
